### PR TITLE
8978 - Homepage featured cards section

### PR DESCRIPTION
**Pull request by anjenkin**
**High level description:**

Homepage featured cards section

**Technical details:**

Technical details for the knowledge of other developers.

**JIRA Ticket:**
[DEV-8978](https://federal-spending-transparency.atlassian.net/browse/DEV-8978)

**Mockup:**
https://app.zeplin.io/project/61f8076f4ef77f11fcc37f55/screen/630cf7444a2af313e4b149c0

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)

Reviewer(s):
- [ ] Code review complete

### CODE REVIEW

File: src/_scss/pages/homepage/homepage.scss [hunk 1]
```diff
@@ -3,6 +3,19 @@
...
+  .flex-gap {
+    display: inline-flex;
+    flex-wrap: wrap;
+  }
+
+  .flex-gap > div {
+    width: 20px;
+    height: 20px;
+    margin: 6px;
+    background-color: lightblue;
```
Comment (nick-torres) [inline src/_scss/pages/homepage/homepage.scss:15]: what is that background color doing?

File: src/js/components/sharedComponents/commonCards/CardContainer.jsx [hunk 2]
```diff
@@ -10,17 +10,29 @@ const propTypes = {
...
-    height: PropTypes.number
+    height: PropTypes.number,
+    className: PropTypes.string
+};
+
+const flexStyles = {
+    display: "flex",
+    "flex-direction": "column",
+    flex: "1 1 auto",
+    "flex-basis": "auto",
```
Comment (nick-torres) [inline src/js/components/sharedComponents/commonCards/CardContainer.jsx:21]: i think this should be flexBasis

Comment (nick-torres) [reply src/js/components/sharedComponents/commonCards/CardContainer.jsx:21]: image

**Comment by nick-torres** [comment]
image
image
image
image

File: src/_scss/pages/homepage/homepage.scss (hunk 1 above)
Comment (anjenkin) [reply src/_scss/pages/homepage/homepage.scss]: removed, though this was for the tempPage with the card usage examples

**Review by nick-torres — APPROVED** [review]

**Pull request merged**